<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>方舟代肝记录工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>761</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>录入信息</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>微博名</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>微信名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>220</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>是</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>已付款</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>220</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>否</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>290</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>内容</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>280</y>
        <width>541</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>430</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>方舟ID</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>信息列表</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>761</width>
        <height>501</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>9</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>计数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>方舟ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>微博名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>微信名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>已付款</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>内容</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>290</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新数据</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>370</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>首页</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>370</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;上一页</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>370</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>下一页&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>370</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>尾页</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>372</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>共10页</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>370</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>跳到</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>370</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>372</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>页</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>370</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>372</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>150</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
